qa/python_models/bls_model_loading/config.pbtxt

```diff
@@ -37,7 +37,7 @@ output [
 
 instance_group [
   {
-    count: 3
+    count: 1
```

Comment:
Why does an instance count > 1 make it unsafe to use? It's not related to parallel instance loading, right?

Reply:
No, I don't think it's related to parallel instance loading. It's just that all the unit tests are executed during the initialize phase, and loading three model instances simultaneously might cause interference between them. For instance, unloading a model in one instance could impact another unit test that relies on that model in other model instances. This is what I meant by unsafely using the model loading API. In the general usage where the `unload` call is not recommended, using multiple instances should be fine.